.github/workflows/generate-benchmarks-s3.yml

```diff
@@ -30,18 +31,20 @@ jobs:
         with:
           role-to-assume: arn:aws:iam::375504701696:role/GitHubBenchmarkRole
           aws-region: us-east-1
-      - name: Generate tpch locally
+      - name: Generate TPC-H locally
         shell: bash
         run: |
           # We run each query once to make sure we don't upload a file if there's a bug that causes a panic.
           cargo run --release --bin tpch -- --formats parquet,vortex -i1
           aws s3 rm --recursive s3://vortex-bench-dev-eu/tpch-sf1/
           aws s3 cp --recursive bench-vortex/data/tpch/1 s3://vortex-bench-dev-eu/tpch-sf1/
+          rm -rf bench-vortex/data/tpch/
 
       - name: Generate clickbench locally
         shell: bash
         run: |
           # We run each query once to make sure we don't upload a file if there's a bug that causes a panic.
           cargo run --release --bin clickbench -- --formats parquet,vortex -i1
           aws s3 rm --recursive s3://vortex-bench-dev-eu/clickbench/
-          aws s3 cp --recursive bench-vortex/data/clickbench s3://vortex-bench-dev-eu/clickbench/
+          aws s3 cp --recursive bench-vortex/data/clickbench_partitioned s3://vortex-bench-dev-eu/clickbench/
```

Comment:
I think this was wrong? IDK how the AWS deals with it

Reply:
yea I think it was wrong